app/components/Analyst/ChangeStatus.tsx

```diff
@@ -0,0 +1,136 @@
+import { useState } from 'react';
+import { graphql, useFragment } from 'react-relay';
+import styled from 'styled-components';
+import statusStyles from 'data/statusStyles';
+import ChangeStatusModal from './ChangeStatusModal';
+
+interface DropdownProps {
+  statusStyles: {
+    primary: string;
+    backgroundColor: string;
+    pillWidth: string;
+  };
+}
+
+const StyledWithdrawn = styled.div`
+  border: none;
+  border-radius: 16px;
+  appearance: none;
+  padding: 6px 12px;
+  height: 30px;
+  margin-bottom: 16px;
+  color: #414141;
+  background-color: #e8e8e8;
+  cursor: default;
+`;
+
+const StyledDropdown = styled.select<DropdownProps>`
+  color: ${(props) => props.statusStyles?.primary};
+  border: none;
+  border-radius: 16px;
+  appearance: none;
+  padding: 6px 12px;
+  height: 30px;
+  margin-bottom: 16px;
+  width: ${(props) => props.statusStyles?.pillWidth};
+  background: ${(props) => props.statusStyles?.backgroundColor}
+    url("data:image/svg+xml;utf8,<svg viewBox='0 0 140 140' width='24' height='24' xmlns='http://www.w3.org/2000/svg'>
+    <g><path d='m121.3,34.6c-1.6-1.6-4.2-1.6-5.8,0l-51,51.1-51.1-51.1c-1.6-1.6-4.2-1.6-5.8,0-1.6,1.6-1.6,4.2 0,5.8l53.9,53.9c0.8,0.8 1.8,1.2 2.9,1.2 1,0 2.1-0.4 2.9-1.2l53.9-53.9c1.7-1.6 1.7-4.2 0.1-5.8z' fill='${(
+      props
+    ) => props.statusStyles?.primary.replace('#', '%23')}'/></g></svg>")
+    no-repeat;
+  background-position: right 5px top 5px;
+
+  :focus {
+    outline: none;
+  }
+`;
+
+const StyledOption = styled.option`
+  color: ${(props) => props.theme.color.text};
+  background-color: ${(props) => props.theme.color.white};
+`;
+
+const ChangeStatus = ({ query }) => {
+  const queryFragment = useFragment(
+    graphql`
+      fragment ChangeStatus_query on Query {
+        applicationByRowId(rowId: $rowId) {
+          id
+          status
+        }
+        allApplicationStatusTypes {
+          nodes {
+            name
+            description
+            id
+          }
+        }
+      }
+    `,
+    query
+  );
+
+  const { allApplicationStatusTypes, applicationByRowId } = queryFragment;
+  const { status } = applicationByRowId;
+  console.log(status);
+
+  const hiddenStatusTypes = ['draft', 'submitted', 'withdrawn'];
+
+  // Filter unwanted status types
+  const statusTypes = allApplicationStatusTypes.nodes.filter(
+    (statusType) => !hiddenStatusTypes.includes(statusType.name)
+  );
+
+  const getStatus = (statusName, statusList) => {
```

Comment:
Nit: We can move this out of the definition of the component since it's a pure function

Reply:
I update that thank you :bow: